service-print/src/main/java/org/oskari/print/util/StyleUtil.java

```diff
@@ -222,38 +228,17 @@ private static PDColor createFillPattern(PDResources resources, int fillPattern,
         }
         return new PDColor(patternName, pattern);
     }
-
-    public static JSONObject getDefaultOskariStyle () {
-        JSONObject json = new JSONObject();
-        // dot
-        JSONObject image = new JSONObject();
-        JSONObject imageFill = new JSONObject();
-        JSONHelper.putValue(imageFill, "color", "#FAEBD7");
-        JSONHelper.putValue(image, "fill", imageFill);
-        JSONHelper.putValue(image, "shape", 5);
-        JSONHelper.putValue(image, "size", 3);
-        JSONHelper.putValue(json, "image", image);
-        // line
-        JSONObject stroke = new JSONObject();
-        JSONHelper.putValue(stroke, "color", "#000000");
-        JSONHelper.putValue(stroke, "width",1);
-        JSONHelper.putValue(stroke, "lineDash", "solid");
-        JSONHelper.putValue(stroke, "lineCap", "butt" );
-        JSONHelper.putValue(stroke, "lineJoin", "mitre");
-        // area
-        JSONObject strokeArea = new JSONObject();
-        JSONHelper.putValue(strokeArea, "color", "#000000");
-        JSONHelper.putValue(strokeArea, "width", 1);
-        JSONHelper.putValue(strokeArea, "lineDash", "solid");
-        JSONHelper.putValue(strokeArea, "lineJoin", "mitre");
-        JSONHelper.putValue(stroke, "area", strokeArea);
-        JSONHelper.putValue(json, "stroke", stroke);
-        JSONObject fill = new JSONObject();
-        JSONHelper.putValue(fill, "color", "#FAEBD7");
-        JSONObject fillArea = new JSONObject();
-        JSONHelper.putValue(fillArea, "pattern", -1);
-        JSONHelper.putValue(fill, "area", fillArea);
-        JSONHelper.putValue(json, "fill", fill);
-        return json;
+    public static Matrix getMatrixForLabel(Coordinate c, PDPrintStyle.LabelAlign align, PDFont font, float size, String label) throws IOException {
+        float x = align.getX();
+        float y = align.getY();
+        switch(align.getAlign()) {
+            case "center":
+                x -=  PDFBoxUtil.getTextWidth(label, font, size) / 2;
+                break;
+            case "left":
+                x += ICON_SIZE/2;
```

Comment:
Why is left offset by ICON_SIZE/2 but center is (the center of the text) at the coordinate ?

Reply:
Accidentally looked that openlayers sets label outside from icon with 'left' align. Tested again and looked examples. openlayers doesn't consider icon size. Removed this and added offsetX for markers